cmd/minikube/cmd/delete.go

```diff
@@ -601,49 +602,99 @@ func killMountProcess() error {
 	return nil
 }
 
+// killProcess takes a path to look for a pidfile(space-separated),
+// it reads the file and converts it to a bunch of pid ints,
+// then it tries to kill each one of them.
+// If no errors were ecnountered, it cleans the pidfile
 func killProcess(path string) error {
 	pidPath := filepath.Join(path, constants.MountProcessFileName)
 	if _, err := os.Stat(pidPath); os.IsNotExist(err) {
 		return nil
 	}
-
 	klog.Infof("Found %s ...", pidPath)
-	out, err := os.ReadFile(pidPath)
+
+	ppp, err := GetPids(pidPath)
 	if err != nil {
-		return errors.Wrap(err, "ReadFile")
+		return err
 	}
-	klog.Infof("pidfile contents: %s", out)
-	pid, err := strconv.Atoi(string(out))
-	if err != nil {
-		return errors.Wrap(err, "error parsing pid")
+
+	// we're giving a change to each pid, to be killed
+	var errs []error
+	for _, pp := range ppp {
+		err := tryKillOne(pp)
+		if err != nil {
+			errs = append(errs, err)
+		}
+
+	}
+
+	if len(errs) == 1 {
+		// if we've encountered only one error, we're returning it:
+		return errs[0]
+	} else if len(errs) != 0 {
+		// if multiple errors were encountered, we're outputting them
+		// and returning a new multiple-errors error:
+		out.Step(style.Failure, "Ecountered multiple errors:")
```

Comment:
```suggestion
		out.Styled(style.Failure, "Multiple errors encountered:")
```
`out.Step` should only be used when a JSON step has been setup for the action.